mne/io/brainvision/brainvision.py

```diff
@@ -429,57 +315,44 @@ def _get_eeg_info(vhdr_fname, eog, misc, response_trig_shift):
 
     # get sampling info
     # Sampling interval is given in microsec
-    sfreq = 1e6 / cfg.getfloat('Common Infos', 'SamplingInterval')
-    sfreq = int(sfreq)
-    eeg_info['n_eeg_chan'] = n_eeg_chan = cfg.getint('Common Infos',
-                                                     'NumberOfChannels')
+    sfreq = int(1e6 / cfg.getfloat('Common Infos', 'SamplingInterval'))
```

Comment:
just wondering, so what's the verdict on `sfreq`? do we store it as an `int` or `float`?